src/analysis/processing/qgsalgorithmrenametablefield.cpp

```diff
@@ -0,0 +1,115 @@
+/***************************************************************************
+                         qgsalgorithmrenametablefield.cpp
+                         -----------------------------------
+    begin                : November 2019
+    copyright            : (C) 2019 by Alexander Bruy
```

Comment:
Header copyright and email needs updating here?

Reply:
Thanks, fixed